sleep-for-days.md

```diff
@@ -0,0 +1,81 @@
+### Go
+---
+```
+// Send an email every 30 days
+func SleepForDaysWorkflow(ctx workflow.Context) (string, error) {
```

Comment:
I think accepting a "user ID" parameter as the old one did makes sense, but not required